packages/dx-react-grid/src/plugins/table-header-row.jsx

```diff
@@ -31,6 +32,9 @@ export class TableHeaderRow extends React.PureComponent {
           connectGetters={(getter, { tableColumn: { column } }) => {
             const sorting = getter('sorting');
             const columns = getter('columns');
+            const sortingScope = getter('tableColumns')
```

Comment:
We already have plans to support showing grouped column data in the table. It will break this feature.